engine.py

```diff
@@ -550,12 +547,20 @@ def _emit_log_message_FUTURE(self, handler, record):
         """
 
         # Give a standard format to the message:
-        #     Shotgun DEBUG <basename>: <message>
+        #     Shotgun DEBUG (<timing>) <basename>: <message>
         # or
         #     Shotgun <basename>: <message>
-        msg = "Shotgun %s%s: %s" % ("DEBUG " if record.levelno < logging.INFO else "",
-                                    record.basename,
-                                    record.message)
+        # where "basename" is the leaf part of the logging record name,
+        # for example "tk-multi-shotgunpanel" or "qt_importer".
+        if record.levelno < logging.INFO:
+            current_time_stamp = time.time()
+            msg = "Shotgun DEBUG (%0.3fs) %s: %s" % (current_time_stamp-self._debug_msg_time_stamp,
```

Comment:
In maya, the output in the script editor looks like this:

```
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Bold.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-CondLight.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Italic.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Light.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Regular.ttf // 
// Shotgun: Toolkit platform was restarted. // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Bold.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-CondLight.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Italic.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Light.ttf // 
```

The debug information should be output consistently, so generate entries on this form:

```
// Debug: Shotgun: Toolkit platform was restarted. // 
```